docs/Q-and-A.md

```diff
@@ -0,0 +1,277 @@
+
+Beware: A Q&A is always *work in progress*. Tips & help welcome.
+
+
+### Disclaimer
+None of the respondents in the **open** repo and presentations have been explicitly named as a source, except for ***Samuel M. Smith Ph.D.*** and ***@henkvancann***. If there is no reference added to the answers, then it's Samuel M. Smith who answered the question. Most of the editing is done by @henkvancann, which might have introduced ommission, errors, language glitches and such. Sorry for that, feel free to correct by submitting a pull request (PR).<br/>
+For practical reasons educational images uploaded by Github members have been downloaded. We de-personalised them by giving images a new name. Under these new names these images have been uploaded to github and used in the Q&A to clarify the questions and answers.
+
+Keri's content is licensed under the [CC by SA 4.0. license](https://creativecommons.org/licenses/by-sa/4.0/). Parts of the video offered on SSI Meetup webinar 58 have been captured and uploaded to Github to support the answers to general questions about digital identity and more in depth answers to question about Keri.
+
+We've done our best to protect the privacy of the Github by investigating the images we used. We haven't come across personal identifiable information (pii). However, should we have made a mistake after all, please let us know and we'll correct this immediately.
+
+### List of questions and definitions
+
+- [Definitions](#definitions)<br/>
+- [Q&A section General](#qa-section-general)
+  * [What is KERI?](#what-is-keri)
+  * [Why use KERI?](#why-use-keri)
+  * [In what programming languages is KERI available?](#In-what-programming-languages-is-keri-available)
+- [Q&A section KERI operational](#qa-keri-operational)
+  * [Where can I download KERI?](#where-can-i-download-keri)
+- [Q&A section Userinterface](#qa-section-userinterface)
+- [Q&A section Root of Trust](#qa-section-root-of-trust)
+- [Q&A section Why the internet is broken](#qa-section-why-the-internet-is-broken)
+- [Q&A section Identifiers](#qa-section-identifiers)
+- [Q&A section Event logs](#qa-section-event-logs)
+- [Q&A section Inconsistency and duplicity](#qa-inconsistency-and-duplicity)
+- [Q&A section Key rotation](#qa-key-rotation)
+- [Q&A section KEL and KERL](#qa-section-kel-and-kerl)
+- [Q&A section Wallets](#qa-section-wallets)
+- [Q&A section Signatures](#qa-section-signatures)
+- [Q&A section Proofs](#qa-section-proofs)
+- [Q&A section Private Key Management](#qa-section-private-key-management)
+- [Q&A section Blockchain](#qa-blockchain)
+- [Q&A section Agencies](#qa-key-agencies)
+- [Q&A section Witness](#qa-section-Witness)
+- [Q&A section Watchers](#qa-section-watcher)
+
+## Knowledge you should be confidently applying
+- The definitions above
+- Public private key pairs
+- Bitcoin Improvement Protocols: BIP32, BIP39, BIP44, BIP47, BIP49, BIP84, BIP174
```

Comment:
What about linking BIPs so it would be easier to navigate to them?

Reply:
good idea, and feel free to do it before me